pr-data.csv

```diff
@@ -1391,14 +1392,15 @@ https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-s
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-streaming-java,org.apache.flink.streaming.runtime.operators.windowing.MergingWindowSetTest.testMergeLargeWindowCoveringMultipleWindows,ID,Rejected,https://github.com/apache/flink/pull/11266,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-connectors/flink-connector-kafka-base,org.apache.flink.table.descriptors.KafkaTest.testValidation,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/8
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-common,org.apache.flink.table.expressions.ExpressionTest.testValueLiteralString,ID,Accepted,https://github.com/apache/flink/pull/11269,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapField,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapField,ID,,,
```

Comment:
Why are you changing PR on this test? If you want to mark that it was moved, then report when it was moved and list the new commit where it's still flaky.